test/test-parse-statement.cpp

```diff
@@ -472,8 +473,9 @@ TEST(test_parse, if_without_body) {
                                       "visit_exit_block_scope"));
     EXPECT_THAT(v.errors,
                 ElementsAre(ERROR_TYPE_FIELD(
-                    error_missing_body_for_if_statement, if_and_condition,
-                    offsets_matcher(&code, strlen(u8"{\n"), u8"if (a)"))));
+                    error_missing_body_for_if_statement, after_if_condition,
+                    offsets_matcher(&code, strlen(u8"{\nif (a)"),
+                                    strlen(u8"{\nif (a)")))));
```

Comment:
Instead of `strlen` for the second offset, you can write the span of characters. In this case, the span is empty:
```suggestion
                    offsets_matcher(&code, strlen(u8"{\nif (a)"),
                                   u8""))));
```